manager/assets/modext/widgets/core/modx.searchbar.js

```diff
@@ -97,25 +110,35 @@ MODx.SearchBar = function(config) {
                     this.tpl.type = null;
                 }
             }
-            ,focus: this.focusBar
-            ,blur: this.blurBar
+            ,focus: {
+                fn: function(){
+                    this.focusBar();
+                }
+            }
+            ,blur: {
+                fn: function(){
+                    this.blurBar()
+                }
+            }
             ,scope: this
         }
+
     });
+
     MODx.SearchBar.superclass.constructor.call(this, config);
-    //this.setKeyMap();
+    this.setKeyMap();
```

Comment:
I worry about this re-opening these: #11794 and #11828 -- however, I also would like this functionality restored. Not sure how to prevent this issue for impacted keyboard regions while still enabling this, unless the shortcut itself is changed to something non-conflicting?